init/module-setup.sh

```diff
@@ -25,6 +25,11 @@ kdump_check_net() {
     elif [ "${KDUMP_NETCONFIG%:force}" != "$KDUMP_NETCONFIG" ]; then
         # always set up network
         kdump_neednet=y
+    elif [ -f "/usr/lib/fence_kdump_send" ] &&
+         ( [[ $KDUMP_PRESCRIPT =~ "fence_kdump_send" ]] || \
+         [[ $KDUMP_POSTSCRIPT =~ "fence_kdump_send" ]] ) ; then
```

Comment:
This test is a bit crude. It would trigger on scripts like `echo "Missing fence_kdump_send"`. OTOH I have no better idea.

Reply:
Agreed.